### PR TITLE
Mobile UI fixes

### PR DESCRIPTION
**Pull request by tuntisz**
- Collapse logo and nav
- Sticky header

Screenshot_2020-04-01 HospitalHero Healthcare workers need our help to fight COVID-19(2)
Screenshot_2020-04-01 HospitalHero Healthcare workers need our help to fight COVID-19(1)

### CODE REVIEW

**Review by cantino — APPROVED** [review]

File: app/javascript/packs/components/Header.jsx [hunk 1]
```diff
@@ -20,32 +20,69 @@ const Logo = () => (
...
-      maxWidth: "30vw",
+      maxWidth: "60vw",
       "& > svg": {
         maxWidth: "100%",
       },
     },
   },
+  [theme.breakpoints.down("sm")]: {
+    sticky: {
+      "& > div > div": {
```
Comment (cantino) [inline app/javascript/packs/components/Header.jsx]: Could something have a `class` set to avoid needing the CSS path?

Comment (tuntisz) [reply app/javascript/packs/components/Header.jsx]: Good call. I wasn't sure how to reference other classes in CSS-in-JS so this was me being lazy. I'll poke at it a bit more - this is very fragile.

File: app/javascript/packs/components/Header.jsx [hunk 2]
```diff
@@ -20,35 +20,62 @@ const Logo = () => (
...
-        <Link className={classes.logo} to="/">
-          <Logo />
+        <Link
+          className={`${classes.logo} ${isScrolled ? classes.hideLogo : ""}`}
+          to="/"
+        >
+          <Logo className={classes.logoSvg} />
         </Link>
-        <Nav />
+        <Nav className={`${isScrolled ? classes.stickyNav : ""}`} />
```
Comment (kpennachio) [inline app/javascript/packs/components/Header.jsx:78]: Oh nice! I was thinking more like sending `isScrolled` as a prop to Nav and then defining the `stickyNav` class in Nav.jsx but this is good!

**Review by kpennachio — APPROVED** [review]

**Review by cantino — APPROVED** [review]
:+1:

**Pull request merged**